### PR TITLE
OSSMDOC-302: Upgrading review

### PR DESCRIPTION
**Pull request by neal-timpe**
- Aligned team: Service Mesh
- for branches 4.6, 4.7, 4.8
- https://issues.redhat.com/browse/OSSMDOC-302
- https://deploy-preview-32953--osdocs.netlify.app/openshift-enterprise/latest/service_mesh/v2x/upgrading-ossm
- SME review: brian-avery
- QE review: gbaufake
- Peer review: mikemckiernan

### CODE REVIEW

**Comment by gbaufake** [comment]
@neal-timpe This one I need to review in special :)

**Comment by neal-timpe** [comment]
@gbaufake This one is still in progress. I've just started. I don't think it's ready for an approval type review yet. 

But if you have feedback on this topic, it would be great if you could leave it here. I'll add you as a reviewer when I have more content.

**Review by gbaufake — APPROVED** [review]

**Review by brian-avery — APPROVED** [review]

File: service_mesh/v2x/upgrading-ossm.adoc [hunk 1]
```diff
@@ -5,7 +5,7 @@ include::modules/ossm-document-attributes.adoc[]
 
 toc::[]
 
-To access the most current features of {ProductName}, upgrade to the current version, 2.0. Upgrading from version 1.1 to 2.0 requires manual steps that migrate your workloads and apps to a new instance of {ProductName} running the new version.
+Upgrading from version 1.1 to 2.0 requires manual steps that migrate your workloads and apps to a new instance of {ProductName} running the new version.
```
Comment (mikemckiernan) [inline service_mesh/v2x/upgrading-ossm.adoc:8]: Just checking that you really want "apps" in the sentence instead of "applications."

**Review by mikemckiernan — APPROVED** [review]
/lgtm

File: service_mesh/v2x/upgrading-ossm.adoc (hunk 1 above)
Comment (neal-timpe) [reply service_mesh/v2x/upgrading-ossm.adoc]: No, I did not mean to do that. Thanks for pointing it out. :)

**Pull request merged**